src/components/__tests__/__fixtures__/sampleFlawComments.ts

```diff
@@ -0,0 +1,78 @@
+import type { ZodFlawCommentType } from '@/types/zodFlaw';
+
+export function sampleFlawComments_1(): ZodFlawCommentType[] {
```

Comment:
```suggestion
export function sampleFlawComments(): ZodFlawCommentType[] {
```

Reply:
Hey @superbuggy I have changed my mind a bit on this point, to a medium preference for permiting an exception on our Camel case notion convention for this sample data cases.

The reason is that removing the underscore makes less clear which number of the sample data it's being used.
For example: `sampleFlawAffects_1` vs `sampleFlawAffects1`, `sampleFlawComments_0` vs `sampleFlawComments0`

We can discuss it in today's code review.